apps/backend/app/base.py

```
@@ -1,10 +1,13 @@
 import os
+import logging
 
 from contextlib import asynccontextmanager
-from fastapi import FastAPI, HTTPException
+from fastapi import FastAPI, HTTPException, Request
```

Ruff (0.11.9)

`fastapi.Request` imported but unused

Remove unused import: `fastapi.Request`

(F401)